src/cow-react/modules/limitOrders/hooks/useCalculateRate.ts

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from 'react'
+import { useAtom } from 'jotai'
+
+import { limitOrdersAtom } from '../state/limitOrdersAtom'
+import { limitRateAtom } from '../state/limitRateAtom'
+
+// Rounds number only if the number of decimals goes above the decimals param
+export const limitDecimals = (amount: number, decimals: number): number => {
```

Comment:
oh nice catch
I will just create separate util function for this but as mentioned previously this will probably be obsolete once we switch to better value types.